webpack.common.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path')
+const { VueLoaderPlugin } = require('vue-loader')
+const StyleLintPlugin = require('stylelint-webpack-plugin')
+
+module.exports = {
+	entry: path.join(__dirname, 'src', 'main.js'),
+	output: {
+		path: path.resolve(__dirname, './js'),
+		publicPath: '/js/',
+		filename: 'files3d.js',
+		chunkFilename: 'chunks/[name].js'
+	},
+	module: {
+		rules: [
+			{
+				test: /\.css$/,
+				use: ['vue-style-loader', 'css-loader']
+			},
+			{
+				test: /\.scss$/,
+				use: ['vue-style-loader', 'css-loader', 'sass-loader']
+			},
+			{
+				test: /\.(js|vue)$/,
+				use: 'eslint-loader',
+				enforce: 'pre'
+			},
+			{
+				test: /\.vue$/,
+				loader: 'vue-loader'
+			},
+			{
+				test: /\.js$/,
+				loader: 'babel-loader',
+				exclude: /node_modules/
+			},
+			{
+				test: /\.(png|jpg|gif|svg)$/,
+				loader: 'url-loader'
+			}
+		]
+	},
+	plugins: [
+		new VueLoaderPlugin(),
+		// new StyleLintPlugin()
+	],
+	resolve: {
+		alias: {
+			Assets: path.resolve(__dirname, 'src/assets/'),
+			Components: path.resolve(__dirname, 'src/components/'),
+			Mixins: path.resolve(__dirname, 'src/mixins/'),
+			Models: path.resolve(__dirname, 'src/models/'),
+			Services: path.resolve(__dirname, 'src/services/'),
+			Views: path.resolve(__dirname, 'src/views/')
+		},
```

Comment:
You copied it from viewer?
I suggest you take inspirations from the simple tutorial app: https://github.com/nextcloud/app-tutorial/blob/master/webpack.common.js

Should be much standard :)

Reply:
I copied it from some other nc app, but that was like 100 years ago :D